README.md

```diff
@@ -22,11 +22,11 @@ Run `node create_release_project.js --version <release_vers>` to create a new pr
 
 EX: `node create_release_project.js --version 3.3.0`
 
-This will generate a project in medic-release-testing repo. Once completed you should see the link to the project in the console. 
+This will generate a project in cht-release-testing repo. Once completed you should see the link to the project in the console. 
 
-EX: Project created at: https://github.com/medic/medic-release-testing/projects/27
+EX: Project created at: https://github.com/medic/cht-release-testing/projects/27
 
-To generate a project with specific test cases (eg for minor releases and patches), add a new label to the issues you need to test and modify labels in [config.js](https://github.com/medic/medic-release-testing/blob/master/config.js#L36) to use the new label.
+To generate a project with specific test cases (eg for minor releases and patches), add a new label to the issues you need to test and modify labels in [config.js](https://github.com/medic/cht-release-testing/blob/main/config.js#L36) to use the new label.
 
```

Comment:
Good job on this one (was a little behind with medic vs cht). Will the [link](https://github.com/medic/cht-release-testing/blob/main/config.js#L36) to `main` work out of the box without renaming `master` > `main` in the repo @jonathanvq ?

Reply:
Hey @ngaruko, it will not work but that's expected. If you check my PR s[teps/tasks](https://github.com/medic/cht-release-testing/pull/195#issue-1181375207) I mention that after this pull request I will rename the branch (step #2), and once it has been renamed I'll merge the changes.